src/components/localization/localized-link.js

```diff
@@ -95,6 +96,7 @@ export const LocalizedLink = React.forwardRef(({ to, ...props }, ref) => {
         is_binary_link,
         is_dbot_link,
         is_deriv_app_link,
+        is_deriv_developer_url,
```

Comment:
```suggestion
        is_deriv_developer_link,
```

Reply:
To be consistent with the rest, this should be `link` instead of `url`. The other references should be changed as well.

Reply:
Very well noted @sean-binary ...acting on it.

Reply:
Very well noted @sean-binary ...acting on it.